src/js/content/npm.js

```diff
@@ -1,7 +1,8 @@
 /* global getBadge */
-const packageName = document.location.pathname.split('/')[2];
-const packageVersion = document.location.pathname.split('/')[4] || 'latest';
-const testPath = `https://snyk.io/test/npm/${packageName}/${packageVersion}`;
+
+var packageName = document.location.pathname.split('/')[2];
+var packageVersion = document.location.pathname.split('/')[4] || 'latest';
+var testPath = `https://snyk.io/test/npm/${packageName}/${packageVersion}`;
```

Comment:
Would be good to stick with `const` over `var` here. Is there particular reason you opted for `var`?

Reply:
because the script runs every time the background onHistoryUpdated fires off and throws errors if using const since it's trying to define the whole thing again. Probably needs to add some logic to check if it doesn't already exist and leave it as const. will update.

Reply:
actually here I think it'll need to be var since we'll re-use this logic and existing variable if the client side nav triggers this.